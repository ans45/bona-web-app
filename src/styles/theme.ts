import { createMuiTheme, ThemeOptions } from '@material-ui/core/styles';

export const themeSettings: ThemeOptions = {
  palette: {
    primary: {
      light: '#6fcdff ',
      main: '#2d9cdb',
      dark: '#006ea9',
      contrastText: '#fff',
    },
  },
  shape: {
    borderRadius: 8,
  },
};

export const theme = createMuiTheme(themeSettings);
