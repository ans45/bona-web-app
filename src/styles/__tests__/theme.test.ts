import { themeSettings } from '../theme';

describe(__filename, () => {
  describe('theme settings', () => {
    it('should match snapshot', () => {
      expect(themeSettings).toMatchSnapshot();
    });
  });
});
