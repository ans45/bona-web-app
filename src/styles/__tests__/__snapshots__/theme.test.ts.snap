// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`theme theme settings should match snapshot 1`] = `
Object {
  "palette": Object {
    "primary": Object {
      "contrastText": "#fff",
      "dark": "#006ea9",
      "light": "#6fcdff ",
      "main": "#2d9cdb",
    },
  },
  "shape": Object {
    "borderRadius": 8,
  },
}
`;
